src/main/java/homework_2/pyramid_printer/PyramidPrinter.java

```diff
@@ -0,0 +1,37 @@
+package homework_2.pyramid_printer;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class PyramidPrinter {
+
+    private int pyramidSize;
+
+    public void run() {
+        System.out.println("Please, input number:");
+        if (validation()) {
+            printPyramid();
+        } else {
+            System.out.println("Only 1 non-negative integer is allowed as passed parameter");
+        }
+    }
+
+    private boolean validation() {
```

Comment:
maybe rename this method for example initAndValidate

Reply:
method names are verbs, according to Code Conventions